README.md

```diff
@@ -46,6 +46,7 @@ Debian system.
 ##### Install (upgrade) required Python libraries
 
 	$ sudo pip install --upgrade trappy bart-py devlib
+	$ sudo pip install netifaces
```

Comment:
Can't this be installed just via apt-get?
This allows to use pip only for main dependency libs...